app/views/posts/_single.html.haml

```diff
@@ -1,5 +1,5 @@
 .well
-  .post
+  .post{id: "post-#{post.id}"}
```

Comment:
There's a PR attempting to get haml-lint into our CI soon. Can you run a haml-linter over it?

```
app/views/posts/_single.html.haml:2 [W] SpaceInsideHashAttributes: Hash attribute should start with one space after the opening brace
app/views/posts/_single.html.haml:2 [W] SpaceInsideHashAttributes: Hash attribute should end with one space before the closing brace
app/views/posts/_single.html.haml:5 [W] RuboCop: Use the new Ruby 1.9 hash syntax.
app/views/posts/_single.html.haml:5 [W] RuboCop: Use the new Ruby 1.9 hash syntax.
app/views/posts/_single.html.haml:5 [W] RuboCop: Use the new Ruby 1.9 hash syntax.
app/views/posts/_single.html.haml:5 [W] LineLength: Line is too long. [84/80]
app/views/posts/_single.html.haml:30 [W] RuboCop: Use the new Ruby 1.9 hash syntax.
app/views/posts/_single.html.haml:32 [W] SpaceBeforeScript: The - symbol should have one space separating it from code
app/views/posts/_single.html.haml:33 [W] RuboCop: Use the new Ruby 1.9 hash syntax.
app/views/posts/_single.html.haml:35 [W] SpaceBeforeScript: The - symbol should have one space separating it from code
app/views/posts/_single.html.haml:42 [W] LineLength: Line is too long. [132/80]
app/views/posts/_single.html.haml:46 [W] LineLength: Line is too long. [115/80]
```